src/common/core/corerel.js

```diff
@@ -1156,18 +1166,22 @@ define([
             aboveAncestor = 1;
 
             var oldNode = node;
-            if (takenRelids) {
-                if (takenRelids[innerCore.getRelid(oldNode)]) {
-                    node = innerCore.createChild(parent, takenRelids, relidLength);
+            if (typeof newRelid === 'string') {
```

Comment:
Shouldn't the same check as in `createNode` be applied here? (I know the newRelid isn't exposed at the top level core)